README.md

```diff
@@ -4,216 +4,311 @@
-|:-----------|:---------------|:----------------------------|:----------------|:----------------------------------|:---------------|:----------------------|
-| ✅          | ✅              | ✅                           | ✅               | ✅                                 | Coming...      | Coming...             |
+One of the most unique and useful things about Semgrep is how easy it is to write and iterate on queries. 
 
-Example patterns:
+The goal is to make it as *easy as possible* to go from an idea in your head to finding the code patterns you intend to.
 
-| **Pattern**                                                        | **Matches**                                                |
-|:-------------------------------------------------------------------|:-----------------------------------------------------------|
-| [`$X == $X`](https://semgrep.live/20B)                             | `if (node.id == node.id): ...`                             |
-| [`requests.get(..., verify=False, ...)`](https://semgrep.live/jqn) | `requests.get(url, timeout=3, verify=False)`               |
-| [`os.system(...)`](https://semgrep.live/1W5)                       | `from os import system; system('echo semgrep')`            |
-| [`$ELEMENT.innerHTML`](https://semgrep.live/9ze)                   | ``el.innerHTML = "<img src='x' onerror='alert(`XSS`)'>";`` |
-| [`$TOKEN.SignedString([]byte("..."))`](https://semgrep.live/rXW)   | `ss, err := token.SignedString([]byte("HARDCODED KEY"))`   |
+**Example**: Let's say you want to find all calls to a function named `exec`, and you don't care about the arguments. With Semgrep, you could simply supply the pattern `exec(...)` and you'd match:
 
-→ [see more example patterns in the live registry viewer](https://semgrep.live/registry)
+~~~python
+# Simple cases grep could find
+exec("ls")
+exec(some_var)
+
+# But you don't have to worry about whitespace
+exec (foo)
+
+# Or calls across multiple lines
+exec (
+    bar
+)
+~~~
+
+Importantly, Semgrep would *not* match the following:
+
+~~~python
+# grep would match this, but semgrep ignores it because 
+# it doesn't have the right function name
+other_exec(bar)
+
+# semgrep ignores commented out lines
+# exec(foo)
+
+# and hard-coded strings
+print("exec(bar)")
+~~~
+
+Semgrep can even do fancy things like matching aliased imports:
+
+~~~python
+# Semgrep knows that safe_function refers to exec so it
+# will still match!
+#   Oof, try finding this with grep
+import exec as safe_function
+safe_function(tricksy)
+~~~
+
+Play with this example in your browser here: https://semgrep.live/QrkD.
+
+For more info on what you can do in patterns, see the [pattern features
+docs](docs/pattern-features.md).
 
 ## Installation
 
 On macOS, binaries are available via [Homebrew](https://formulae.brew.sh/formula/semgrep):
 
 ```bash
-brew install returntocorp/semgrep/semgrep
+$ brew install returntocorp/semgrep/semgrep
```

Comment:
pet peeve but I'd suggest removing this $ and the one blow, if you know what brew is you don't need a $ reminder and it is always annoying to not be able to triple-click and paste

Reply:
Sure, happy to do that! To me, the `$` is a nice signal that you're at a terminal vs its a code snippet, but agreed re: it makes it annoying to copy/paste. And if you're reading about Semgrep, you're probably comfortable with the terminal.